sequencer/src/api/sql.rs

```diff
@@ -580,10 +605,12 @@ async fn load_frontier<Mode: TransactionMode>(
 }
 
 async fn load_v1_reward_accounts<Mode: TransactionMode>(
+    db: &SqlStorage,
     tx: &mut Transaction<Mode>,
```

Comment:
I think it is safe to remove this tx parameter and open a new read transaction inside this function to get the leaf